test/e2e/config/ironic.yaml

```diff
@@ -23,7 +23,8 @@ variables:
   # Test credentials. The tests will create a BMH with these.
   BMC_USER: admin
   BMC_PASSWORD: password
-  BMC_ADDRESS: "ipmi://192.168.222.1:16230"
+  # BMC_ADDRESS: "ipmi://192.168.222.1:16230"
```

Comment:
Hmm true. It is not really needed but works as example. In fact all the BMC variables here are overridden by environment variables so they are not needed in the file. However, I find it useful to keep them here as an example

Reply:
I did a solution now with commenting the use of it. I think it can be good to save them for a while at least during development. Hope you like the solution.

Reply:
Thanks.
/approved